token/client/src/token.rs

```diff
@@ -894,6 +916,88 @@ where
         .await
     }
 
+    /// Wrap lamports into native account
+    pub async fn wrap<S: Signers>(
+        &self,
+        account: &Pubkey,
+        owner: &Pubkey,
+        lamports: u64,
+        signing_keypairs: &S,
+    ) -> TokenResult<T::Output> {
+        // mutable owner for Tokenkeg, immutable otherwise
+        let immutable_owner = self.program_id != spl_token::id();
+        let instructions = self.wrap_ixs(account, owner, lamports, immutable_owner)?;
+
+        self.process_ixs(&instructions, signing_keypairs).await
+    }
+
+    /// Wrap lamports into a native account that can always have its ownership changed
+    pub async fn wrap_transferable<S: Signers>(
```

Comment:
nit: how about full clarity with the name and say `wrap_with_mutable_ownership`? `transferable` is ambiguous because of the non-transferable token extension